hypervisor/src/cpu.rs

```diff
@@ -115,6 +115,16 @@ pub enum HypervisorCpuError {
     #[error("Failed to get Saved Processor Extended States: {0}")]
     GetXsaveState(#[source] anyhow::Error),
     ///
+    /// Getting the VP state components error
+    ///
+    #[error("Failed to get VP State Components: {0}")]
+    GetAllVpStateComponents(#[source] anyhow::Error),
+    ///
+    /// Setting the VP state components error
+    ///
+    #[error("Failed to set VP State Components: {0}")]
+    SetAllVpStateComponents(#[source] anyhow::Error),
```

Comment:
Can you put them behind the `mshv` feature gate given they are mshv specific?

Reply:
Of course, feel free to put `Set/GetXsaveState` error behind `kvm` feature gate if they are not going to be used by `mshv` by design.

Reply:
Thank you. Addressed your comments.